tests/e2e/utils/flows.js

```diff
@@ -85,9 +84,47 @@ export const AdminFlow = {
 		await CommonFlow.login( 'admin', 'password' );
 	},
 
-	goToPlugins: async () => {
-		await page.goto( baseUrl + '/wp-admin/plugins.php', {
+	goTo: async ( url ) => {
+		await page.goto( adminUrl( url ), {
 			waitUntil: 'networkidle2',
 		} );
 	},
+
+	goToPlugins: async () => {
+		return AdminFlow.goTo( 'plugins.php' );
+	},
+
+	async findPlugin( slug ) {
```

Comment:
Should this just return the `tr` ir be rebaned findPluginDeactivateLink? Half shared below in the `activatePlugin` helper.

Reply:
Updated in 1ceb850